terraform/Security

```diff
@@ -0,0 +1 @@
+Add the SSH key to your Digital Ocean account under Settings
```

Comment:
What is this file?

Reply:
I have no idea how it got there.  Removed.